src/Features/Core/Portable/DocumentationComments/AbstractDocumentationCommentFormattingService.cs

```diff
@@ -59,6 +79,65 @@ public void AppendParts(IEnumerable<TaggedText> parts)
                 _anyNonWhitespaceSinceLastPara = true;
             }
 
+            public void PushList(DocumentationCommentListType listType)
+            {
+                _listStack.Add((listType, 0, false));
```

Comment:
named params would likely help here.  Also, it's not clear ot me what "renderedItem" means in this entire walk.